documentation/misc/RELEASE_ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,103 @@
+> Release Issue Template
+
+# Lotus X.Y.Z Release
+
+We're happy to announce Lotus X.Y.Z...
+
+## 🗺 What's left for release
```

Comment:
Is this a list of issues that **must** be completed for this release? if so, can we restrict that no more tickets can be added X days before the `Estimated shipping date`, for which i would suggest 7 for X(around when the rc is suppose to be cut)

Reply:
Yeah, I would honestly like to get rid of this section. Thoughts?

I don't want "must be completeds" except in extraordinary situations.

Reply:
"must dos" may be common for state upgrade releases which have an agreed-upon FIP across all implementations. Imagine that should be included here